Baseline interval check (#361)
mne-tools/mne-bids-pipeline: docs/source/changes.md

  ({{ gh(322) }} by {{ authors.hoechenberger }})
- Warn if using ICA and no EOG- or ECG-related ICs were detected.
  ({{ gh(351) }} by {{ authors.crsegerie }})

> - Check that the baseline interval is compatible with [epochs_tmin, epochs_tmax] in the user's configuration file.

Reviewer: ```suggestion
- Check that the baseline interval falls into `[epochs_tmin, epochs_tmax]`.
```
